src/templates/blog-post.js

```diff
@@ -35,19 +54,77 @@ class BlogPostTemplate extends React.Component {
     return (
       <Layout location={location}>
         <Wrapper>
-          <Helmet
-            title={`${frontmatter.title} | ${data.site.siteMetadata.title}`}
-          />
-          <PostHeader postdate={frontmatter.date} />
-          <h1>{frontmatter.title}</h1>
+          <Header/>
+          <div className='Middle'>
+            <div className='HeadTop'>
+            <Helmet
+              title={`${frontmatter.title} | ${data.site.siteMetadata.title}`}
+              
+
+              >
+                <meta itemprop="name" content={`${frontmatter.title}`} />
+                <meta
+                  itemprop="description"
+                  content={`${frontmatter.title} | ${data.site.siteMetadata.title}`}
+                />
+                <meta
+                  itemprop="image"
+                  content="https://reactviz.holiday/reactvizholiday.png"
+                />
+
+                <meta property="og:url" content="https://reactviz.holiday" />
```

Comment:
For specific posts we want all this meta data to be for the post, not the site as a whole.